vita3k/config/include/config/config.h

```diff
@@ -1,5 +1,5 @@
 // Vita3K emulator project
-// Copyright (C) 2018 Vita3K team
+// Copyright (C) 2020 Vita3K team
```

Comment:
happy new year

if we aren't going to change the date for every file in the project its probably okay to leave it

Reply:
I have change only on file changed ^^